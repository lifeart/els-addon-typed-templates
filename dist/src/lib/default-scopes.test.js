"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const default_scopes_1 = require("./default-scopes");
describe('defaultScopes', () => {
    it('return expected list of objects', () => {
        expect(Object.keys(default_scopes_1.defaultScopes())).toMatchSnapshot();
    });
});
//# sourceMappingURL=default-scopes.test.js.map