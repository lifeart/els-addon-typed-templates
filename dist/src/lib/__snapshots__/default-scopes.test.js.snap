// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultScopes return expected list of objects 1`] = `
Array [
  "globalScope",
  "pathsForGlobalScope",
  "tailForGlobalScope",
  "definedScope",
]
`;
